torch/_refs/__init__.py

```diff
@@ -1447,11 +1447,26 @@ def le(a: TensorLikeType, b: TensorLikeType) -> TensorLikeType:
 )
 def logaddexp(a: TensorLikeType, b: TensorLikeType) -> TensorLikeType:
     # Nb. this implementation does nto distribute the gradients evenly when a == b
```

Comment:
mind fixing this typo (nto) too while touching this file? :P

Reply:
Sure, no prob